src/lumigo_opentelemetry/libs/general_utils.py

```diff
@@ -43,3 +52,16 @@ def safe_get_list(lst: list, index: Union[int, str], default=None):
     if not isinstance(lst, Iterable):
         return default
     return lst[index] if len(lst) > index else default
+
+
+def get_max_size() -> int:
```

Comment:
Can you add unit-tests for this function as well? We try to have tests for every public function :)